Posts/2021/02/XML-NewEmployees.md

```diff
@@ -0,0 +1,180 @@
+```yaml
+---
+post_title: 'Leveraging XML with New Employee PowerShell Scripts'
+username: s-r-turner
+categories: ActiveDirectory, XML, PowerShell
+tags: XML, ActiveDirectory
+featured_image:
+summary: using xml referencing to bolster up your new employee scripts with information.
+---
+```
+
+# Leveraging XML With New Employee PowerShell Scripts
+
+I'm going to show how you can leverage XML files for referencing information, to help bolster up your Active Directory user accounts.
+
+## Scenario
+
+You have multiple subsidiary companies that each have their own individual sets of offices and addresses. You need to ensure that each new user you create has the correct address and contact information based on where they are working.
+
+## Getting Started - Creating the XML
+
+To create the xml, we're going to use the XmlWriter class, put this into a variable to build upon and add some formatting options. Like so:
+
+```powershell
+$XMLPath = "C:\Users\sam\AD-References.xml"
+$XMLWriter = New-Object System.XML.XMlTextWriter($XMLPath,$null)
```

Comment:
perhaps add a reference to what an XMLTextWriter is and what you are going to do with it. Also possibly add a URL reference to  [XMLTextWriterClass](https://docs.microsoft.com/en-us/dotnet/api/system.xml.xmltextwriter?view=net-5.0)